File: c++/baekjoon/2588/2588.cpp
#include <stdio.h>

int main()
{
    int num1, num2, temp;

    scanf("%d\n %d", &num1, &num2);

    temp = num2;
    for (int i = 0; i < 3; i++)
    {
        printf("%d\n", num1 * (temp % 10));
        temp /= 10;
    }

    printf("%d\n", num1 * num2);

    return 0;
}

File: python/baekjoon/2588.py
num1 = int(input())
num2 = int(input())

one = num2 % 10
ten = num2 % 100 // 10
hundread = num2 // 100

print(num1 * one)
print(num1 * ten)
print(num1 * hundread)
print(num1 * num2)
